web-server/deploy.example.bash

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 
 if [ "$1" == "" ]; then
-    USER=pbench
+    USER=gkalsi
 else
     USER=$1
 fi
 if [ "$2" == "" ]; then
-    HOST=archivehost.example.com
+    HOST=dhcp31-187.perf.lab.eng.bos.redhat.com
```

Comment:
I don't think we want to check this in hard-coded, but instead use it as a parameter.  We'll want to re-work this commit to pull that reference.